packages/utils/src/lib/value.ts

```diff
@@ -30,7 +30,11 @@ export function isNonNullish<T>(
 	return value !== null && value !== undefined
 }
 
-/** @public */
+/**
+ * Create a deep copy of a value. Uses the structuredClone API if available, otherwise uses JSON.parse(JSON.stringify()).
+ *
+ * @param i - The value to clone.
+ * @public */
```

Comment:
@steveruizok is it worth using core-js polyfill here?

https://github.com/zloirock/core-js#structuredclone